### DISCUSSION TITLE
Add markdown preview to the file editor

### DISCUSSION THREAD

**Discussion by o1egl** [filebrowser/filebrowser#1240] [Ideas]
It would be nice to have a button in the file editor to preview the markdown files

**Comment by viriatvx** [comment]
This would be a great feature for note-taking and bookmarks, is there anything a no coder can do? like donate or something?

**Reply by yjbdsky** [reply]
https://github.com/filebrowser/filebrowser/pull/1127  + edit button + files tree = perfect  note manager

- edit note
- share note

**Reply by yjbdsky** [reply]
#750
#1244
#2109

**Comment by jniggemann** [comment]
This would be great indeed!

**Comment by gittynick** [comment]
I would also love to have a markdown editor
Is it possible to integrate something like Zenpen inside filebrowser ? https://github.com/tholman/zenpen ? 
That would be neat ;-)

**Comment by NChaves** [comment]
It would be great if the editor supported syntax highlighting, section folding, advanced editing and code formatting. Perhaps something like Monaco editor below.
Unfortunately I am not one that can collaborate in a productive way.

https://github.com/microsoft/monaco-editor

I do appreciate the great work the developers have put into filebrowser, thank you.
If there is a way I can contribute, please let me know.

**Comment by StudentWeis** [comment]
I need it!

**Comment by Bojun-Feng** [comment]
That would be a great feature!
